src/client/components/Forms/SignUp.js

```diff
@@ -1,54 +1,166 @@
+          />
+        </label>
+        <label htmlFor="position">
+          Position
+          <input
+            type="text"
+            name="position"
+            placeholder=""
+            value={position}
+            onChange={handlePositionInput}
+            required
+          />
+        </label>
+        <br />
+        <label htmlFor="email">
+          Email
+          <input
+            type="email"
+            name="email"
+            placeholder=""
+            value={email}
+            onChange={handleEmailInput}
+            required
+          />
+        </label>
+        <label htmlFor="img">
+          Profile Picture
+          <input
+            type="url"
+            name="img"
+            placeholder="profile image link"
+            value={image}
+            onChange={handleImageInput}
+            required
+          />
+        </label>
+        <label htmlFor="linkedin">
+          LinkedIn
+          <input
+            type="url"
+            name="linkedin"
+            placeholder="LinkedIn profile link"
+            value={linkedin}
+            onChange={handleLinkedinInput}
+            required
+          />
+        </label>
+        <label htmlFor="github">
+          Github
+          <input
+            type="url"
+            name="github"
+            placeholder="Github profile link"
+            value={github}
+            onChange={handleGithubInput}
+            required
+          />
+        </label>
+        <br />
+        <label htmlFor="password">
+          Password
+          <input
+            type="password"
+            name="password"
+            placeholder=""
+            value={password}
+            onChange={handlePasswordInput}
+            pattern="(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{6,}"
+            title="Must contain at least one number and one uppercase and lowercase letter, and at least 6 or more characters"
+            required
+          />
+        </label>
+        <br />
+        <label htmlFor="password">
+          Password-confirm
+          <input
+            type="password"
+            name="password"
+            placeholder="Write your password again"
+            value={passwordConfirm}
+            onChange={handlePasswordConfirmInput}
+            pattern="(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{6,}"
+            title="Must contain at least one number and one uppercase and lowercase letter, and at least 6 or more characters"
+            required
+          />
+        </label>
+
+        <label className="label-checkbox">
+          <input type="checkbox" id="check" disabled=""></input>
+          Create an account means you're ok with our Terms of Service, Privacy,
+          Policy and our default Notification Settings
+        </label>
+        <br />
+
+        <Button
+          buttonName="Create Account"
+          style={{ backgroundColor: '#da532c' }}
+          type={'submit'} //  type can be change according to button component
```

Comment:
```suggestion
          type="submit" //  type can be change according to button component
```

Reply:
in button component Shruti changes type as a props.